src/hooks/use-expiration-date.ts

```diff
@@ -5,12 +5,28 @@ import { useI18nData } from 'src/hooks/use-i18n-data';
 import { formatDistance } from 'src/lib/date';
 import { SupportedLocale } from 'src/lib/locale';
 
-function formatStringDate( ms: number, locale: SupportedLocale ): string {
-	const formatter = new Intl.DateTimeFormat( locale, {
-		day: 'numeric',
-		month: 'long',
-		year: 'numeric',
-	} );
+function formatStringDate(
+	ms: number,
+	locale: SupportedLocale,
+	format: 'long' | 'short' = 'short'
+): string {
+	const options: Intl.DateTimeFormatOptions =
```

Comment:
Another alternative could be using the format function from `date-fns`. Something like: `format( ms, 'PPpp', { locale: dateFnsLocales[ locale ] } )`

`Intl.DateTimeFormatOptions` is also good!, and consistent with what we had before.

Reply:
Is there anything against using the solution we followed in previous UI?

Reply:
If you mean using `Intl.DateTimeFormatOptions`, there is nothing bad about using it, let's keep it.